watch-package.js

```diff
@@ -57,8 +59,17 @@ module.exports = function watchPackage (pkgDir, exit) {
       return a.concat(b)
     })
 
+    if (ignores) {
+      ignores = [].concat(ignores).map(function (ignore) {
+        return ['--ignore', ignore]
```

Comment:
Good point. A quick search seems to indicate that `child_process.spawn` arguments do not undergo shell expansion, so it seems you're right, and the `--ignore` values don't need to be quoted.